### PR TITLE
[Term Entry] PHP Arrays shuffle()

### LINKED ISSUE #2822: [Term Entry] PHP Arrays shuffle()
### Reason/inspiration (optional)

We would like a new term entry for [PHP](https://www.codecademy.com/resources/docs/php) about the `shuffle()` function.

The entry should go in a new file in `docs/content/php/concepts/arrays/terms/shuffle/shuffle.md` 

The entry should 

- Have a syntax section illustrating the function with a ` ```pseudo ` code block. 
- An example showing the function in use with a ` ```php ` code block. 
- A runnable codebyte example in a ` ```codebyte/php` block. 

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by MMMMMMMZ**
### Description

Added The PHP Array function shuffle() which resolves #2822 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/shuffle/shuffle.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Syntax
+
+```pseudo
+shuffle($array)
+```
+
+The `shuffle()` function has only one parameter:
+
+- `$array`: Specifies the array to randomize Its elements.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/shuffle/shuffle.md:25]:
```suggestion
- `$array`: Specifies the array to randomize.
```

File: content/php/concepts/arrays/terms/shuffle/shuffle.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+shuffle($numArray);
+print_r($numArray);
+?>
+
+```
+
+The example will result in the following output:
+
+> **Note:** The Numbers Might Be arranged differently.
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/shuffle/shuffle.md:46]:
```suggestion
> **Note:** The numbers might be arranged differently.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@MMMMMMMZ I did a first review and left a couple of comments.

**Review by SSwiniarski — APPROVED** [review]
@MMMMMMMZ Looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@MMMMMMMZ @SSwiniarski Overall in solid shape, I just pushed a few minor edits. Looks good to merge

**Comment by SSwiniarski** [comment]
@MMMMMMMZ Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/arrays/shuffle

image

**Pull request merged**